frontend/src/clients/ReportClient.ts

```diff
@@ -11,16 +32,25 @@ export class ReportClient extends HttpClient {
     },
   }
 
-  generateReporter = async () => {
+  reporting = async (params: generateReportParams) => {
+    // eslint-disable-next-line no-useless-catch
     try {
-      await this.axiosInstance.post(`/report`).then((res) => {
-        this.reportResponse = res.data
-      })
+      await this.axiosInstance
+        .post(
+          `/report`,
+          {},
+          {
+            headers: {
+              'content-type': 'application/x-www-form-urlencoded',
+            },
+            params: params,
+          }
+        )
+        .then((res) => {
+          this.reportResponse = res.data
+        })
     } catch (e) {
-      const code = (e as AxiosError).response?.status
-      if (code === HttpStatusCode.InternalServerError) {
-        throw new InternalServerException('report', 'Internal server error')
-      }
+      throw e
```

Comment:
How to handle error case

Reply:
Unified exception handling is performed in Axios response interception, and the corresponding error message is output in the self written effect